devel/sn-demangler/Portfile

```diff
@@ -3,9 +3,10 @@
 PortSystem          1.0
 PortGroup           github 1.0
 PortGroup           java 1.0
+PortGroup           legacysupport 1.0
 
 github.setup        indoorvivants sn-demangler 0.0.3 v
-revision            0
+revision            1
```

Comment:
since you're fixing a build issue there is no need to increase the `revision` since this change doesn't do anything for systems where the build did work correctly.